lib/timers.js

```diff
@@ -438,7 +438,7 @@ function setTimeout(callback, after, arg1, arg2, arg3) {
   return timeout;
 }
 
-setTimeout[internalUtil.promisify.custom] = function(after, value) {
+setTimeout[internalUtil.promisify.custom] = function setTimeoutPromise(after, value) {
```

Comment:
@trott Even for these? A quick experiment in the REPL seems to indicate otherwise…

Reply:
Yeah, that probably needs a name. :-D

Honestly, at this point, the exact "does it need a name or not" determination is fairly convoluted and probably V8-specific. I'd be for us scrapping our entire conversation about it in that other issue and simply enforcing the ESLint `func-names` rule with the `as-needed` option set. But that would require a whole other conversation because there would be changes definitely not in line with people's opinions in the previous conversation.